### PR TITLE
Add 'cppast' package

### PR DESCRIPTION
**Pull request by Bjoe**
* I've followed [this guide](https://cpp-pm-hunter.readthedocs.io/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status _The job exceeded the maximum log length, and has been terminated._:

  * https://travis-ci.org/Bjoe/hunter/builds/628738058
  * https://ci.appveyor.com/project/Bjoe/hunter-t1t06/builds/29722462

* I have submitted CI configs to https://github.com/cpp-pm/hunter-testing targeting `pkg.template` branch,
  see this merged pull request https://github.com/cpp-pm/hunter-testing/pull/26

How can I fix _maximum log length_?

### CODE REVIEW

File: cmake/projects/cppast/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    cppast
+    VERSION
+    0.0.1-dbbbe62
+    URL
+    "https://github.com/Bjoe/cppast/archive/dbbbe626e4cff9da83b1c0baba650fdd56df0cdc.zip"
```
Comment (Bjoe) [inline cmake/projects/cppast/hunter.cmake]: I need to have a repo in cpp-pm

File: cmake/find/Findtiny-process-library.cmake [hunk 2]
```diff
@@ -1,58 +0,0 @@
-# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```
Comment (Bjoe) [inline cmake/find/Findtiny-process-library.cmake]: Unfortunately, I forgot to delete this find module in the last PR.

File: cmake/projects/cppast/hunter.cmake (hunk 1 above)
Comment (bkotzz) [reply cmake/projects/cppast/hunter.cmake]: https://github.com/cpp-pm/cppast

File: cmake/projects/cppast/hunter.cmake [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    cppast
+    VERSION
+    0.0.1-dbbbe62
+    URL
+    "https://github.com/cpp-pm/cppast/archive/dbbbe626e4cff9da83b1c0baba650fdd56df0cdc.zip"
```
Comment (rbsheth) [inline cmake/projects/cppast/hunter.cmake]: Please change the archive URL to https://github.com/cpp-pm/cppast/archive/v0.0.0-b155d6a-p0.tar.gz and the version to `0.0.0-b155d6a-p0`. The version is based on the upstream hash, not the hash of the hunterization commit.

Comment (Bjoe) [reply cmake/projects/cppast/hunter.cmake]: Ok, thanks for the hint. Done.

**Comment by rbsheth** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.23.241

**Comment by Bjoe** [comment]
Ok, thanks for merging .... for the sake of completeness here the builds from the last commit:

I've tested this package remotely and have excluded all broken builds.
Here is the links to the Travis/AppVeyor with status The job exceeded the maximum log length, and has been terminated.:

* https://ci.appveyor.com/project/Bjoe/hunter-t1t06/builds/29938377
* https://travis-ci.org/Bjoe/hunter/builds/633615253

**Comment by rbsheth** [comment]
@Bjoe Found this in the docs: https://hunter.readthedocs.io/en/latest/creating-new/create/cmake.html#fix-travis-log-too-long-error

**Pull request merged**